package.json

```diff
@@ -29,7 +29,7 @@
     "lint:scss": "stylelint 'src/**/*.scss' --fix",
     "prepublishOnly": "npm-run-all -s test:ci lint build lib",
     "prepare": "npm run snyk-protect",
-    "test": "react-scripts test --env=jsdom --transformIgnorePatterns \"node_modules/(?!your-module-name)/\"",
```

Comment:
@rashidakanchwala this was causing some issue. I think you introducded it so I reverted it back.